README.md

```diff
@@ -5,17 +5,34 @@ Minio is an object storage server released under Apache License v2.0. It is comp
 Minio server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL.
 
 ## Docker Container
+
 ### Stable
+
+```sh
+docker run -p 9000:9000 --name minio -d minio/minio server /export
+```
```

Comment:
I think at least the single command needs to be more verbose to be useful - especially with the `-d` flag for those who are new to Docker, it will be confusing to have Minio running in the foreground. We're very familiar with the CLI etc, but think about the range of Minio users.